agent/bootstrap.sh

```diff
@@ -66,6 +66,15 @@ CFLAGS='-g -O0 -ftrapv' meson build \
       -Dnobody-user=nfsnobody \
       -Dnobody-group=nfsnobody
 ninja-build -C build
+
+# Let's check if the new systemd at least boots before installing it
+# As the CentOS' systemd-nspawn version is too old, we have to use QEMU
```

Comment:
In theory the test should be run with `systemd-nspawn` that has just been built. If it isn't, then something is wrong with the test. Could you open an issue?

Reply:
Interesting, you're right. This was a quick guess without any further investigation, as the job was dying because of an unknown property:

```
Press ^] three times within 1s to kill container.
Failed to allocate scope: Cannot set property CollectMode, or unknown property.
Parent died too early
E: nspawn failed with exit code 1
```

Looking at the `test/test-functions` it looks the problem is somewhere else.

Reply:
That looks suspiciously familiar. I hit something like that and even tried to "fix" it in https://github.com/systemd/systemd/pull/9750. As far as I understand, it's "not-a-bug" :-)

Reply:
In that case I'd say it would be safer to just stick with QEMU, at least for this part of the CI.